client-app/xapi/graphql/vendor/mutations/createCustomerReview/index.ts

```diff
@@ -0,0 +1,41 @@
+import globals from "@/core/globals";
+import { useUser } from "@/shared/account/composables";
+import mutationDocument from "./createCustomerReview.graphql";
+import type {
+  CreateCustomerReviewCommandType,
+  Mutations,
+  MutationsCreateCustomerReviewArgs,
+  CustomerReview,
+} from "@/xapi/types";
+
+export type CreateCustomerReviewCommandArgumentsType = Omit<
+  CreateCustomerReviewCommandType,
+  "storeId" | "userId" | "userName"
+>;
+
+export async function createCustomerReview(
+  commandArguments: CreateCustomerReviewCommandArgumentsType
+): Promise<CustomerReview> {
+  const { storeId, userId } = globals;
+
+  const { user } = useUser();
```

Comment:
It shouldn't be here

Reply:
But I need to pass user name (full real name, not email). So what you will propose? Pass it as parameter? Why?